template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPageWithPageModel/.template.config/template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "ASP.NET"],
+  "name": "Razor Page with Page Model",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorPageWithPageModel",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorPageWithPageModel",
+  "shortName": "page",
+  "sourceName": "Index",
+  "primaryOutputs": [ { "path": "Index.cshtml"}, { "path": "Index.cs" } ],
+  "defaultName": "Index",
+  "symbols": {
+    "namespace": { 
```

Comment:
It's probably good to add a description to the namespace symbol, it'll get displayed in the detailed help for the template in the CLI.